src/Optical/Emitters/AngularPower.jl

```diff
@@ -33,7 +33,7 @@ Cosine power distribution. Ray power is calculated by:
 struct Cosine{T} <: AbstractAngularPowerDistribution{T} 
     cosine_exp::T
 
-    function Cosine(cosine_exp::T = one(T)) where {T<:Real}
+    function Cosine(cosine_exp::T = one(Float64)) where {T<:Real}
```

Comment:
calling `Cosine()` didn't provide any information for type inference - we need to give a default type

Reply:
@alfredclwong this is correct but could be confusing to new users. Maybe break into two constructors?

function Cosine(cosine_exp::T) where {T<:Real} = new{T}(cosine_exp)
function Cosine() = new{T}(one(Float64))

It's a small point, if you want to leave it then I'll approve anyway.